examples/demo.js

```diff
@@ -188,8 +188,10 @@ document.addEventListener('DOMContentLoaded', function () {
             })
         })
 
-
-
+        $('.org-wide-issues').on('click', ()=>{
+            location.href = $('.org-wide-issues').attr("href")
+            location.reload();
+        })
```

Comment:
This works perfectly @on2onyekachi!! Can you just make sure that the indentation is two spaces deep? Thank you!!

Reply:
Thanks, @TildaDares No, I followed the indentation of the file, all 4spaces. Should I fix all of it, or make it an FTO?

Reply:
You know what let’s leave it that way so it doesn’t look odd.

Reply:
OKAYY